Team8923_CENTERSTAGE/src/main/java/org/firstinspires/ftc/team8923_CENTERSTAGE/BaseAutonomous.java

```diff
@@ -1,32 +1,19 @@
 package org.firstinspires.ftc.team8923_CENTERSTAGE;
 
-import com.qualcomm.hardware.bosch.BNO055IMU;
 import com.qualcomm.robotcore.hardware.DcMotor;
 import com.qualcomm.robotcore.util.ElapsedTime;
-import com.qualcomm.robotcore.util.Range;
 
 abstract public class BaseAutonomous extends BaseOpMode {
 
     private ElapsedTime runtime = new ElapsedTime();
 
-    int lastEncoderFL = 0;
```

Comment:
Why are you deleting stuff from 8923?

Reply:
Yeah...we have no idea how that happened. However, we have coordinated with 8923 and it has been resolved as this change happened in our branch and didn't affect them. 😊